File: dev/JAVA/Divisor.java
import java.util.Scanner;

public class Divisor {

	public static void main(String[] args) {
		
		int numero1 = 0;
		int numero2 = 0;
		int resultado;

		Scanner reader = new Scanner(System.in);
		
		System.out.println("Introduce el primer número:");			
		numero1 = reader.nextInt();
			
		System.out.println("Introduce el segundo número:");
		numero2 = reader.nextInt();
			
		// Dividimos
		resultado = numero1/numero2;
		
		System.out.println("La división es " + numero1 + " / " + numero2 + " = " + resultado);
		
		// Cerramos el stream de lectura
		reader.close();
			
	}

}

File: dev/C/Divisor.c
#include <stdio.h>
#include <stdlib.h>

int main() {


  int numero1;
  int numero2;
  
  printf("Introduce un numero: ");
  scanf("%d", &numero1);
  printf("Introduce otro numero: ");
  scanf("%d", &numero2);
  
  printf("La division de %d/%d = %d", numero1, numero2, (numero1/numero2)); 
  system("PAUSE");
  return 0;

}